packages/enzyme-utilities/src/test/index.test.tsx

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
 import {mount} from 'enzyme';
-import {trigger} from '..';
-import {Toggle} from './Toggle';
-import {ActionList, Action} from './Actions';
+
+import {trigger, findById} from '..';
+
+import {Toggle} from './fixtures/Toggle';
+import {ActionList, Action} from './fixtures/Actions';
```

Comment:
I don't love having these in separate files anyways, but IMO it was quite messy to have them alongside the test file like this.